src/frontend/components/DashboardVideoLive/index.tsx

```diff
@@ -104,6 +131,61 @@ export const DashboardVideoLive = ({ video }: DashboardVideoLiveProps) => {
   const isLiveStarted =
     video.live_state !== undefined && video.live_state !== liveState.IDLE;
 
+  const lastArraySize = useRef(0);
+  const lastToastId = useRef<Nullable<string>>(null);
+
+  useEffect(() => {
+    if (
+      isLiveStarted &&
+      video.participants_asking_to_join.length !== 0 &&
+      video.join_mode !== JoinMode.FORCED &&
+      lastArraySize.current <= video.participants_asking_to_join.length &&
+      !(isPanelVisible && currentItem === LivePanelItem.VIEWERS_LIST)
+    ) {
+      if (lastToastId.current) {
+        toast.remove(lastToastId.current);
+      }
+      lastToastId.current = toast.custom(
+        <Box
+          background="bg-info"
+          direction="column"
+          gap="small"
+          round="xsmall"
+          pad="medium"
+        >
+          <Text color="blue-active" size="0.875rem">
+            {generateOnStageRequestNotificationMessage(
+              video.participants_asking_to_join,
+              intl,
+            )}
+          </Text>
+
+          <Button
+            primary
+            label={intl.formatMessage(messages.manageRequestBtnLabel)}
+            onClick={() => {
+              setPanelVisibility(true, LivePanelItem.VIEWERS_LIST);
+              toast.dismiss(lastToastId.current!);
+              lastToastId.current = null;
+            }}
+            size="small"
+          />
+        </Box>,
+        {
+          duration: Infinity,
+        },
+      );
+    }
+    lastArraySize.current = video.participants_asking_to_join.length;
+    if (
+      lastToastId.current &&
+      isPanelVisible &&
+      currentItem === LivePanelItem.VIEWERS_LIST
+    ) {
+      toast.remove(lastToastId.current);
+    }
+  }, [video.participants_asking_to_join]);
```

Comment:
(answering https://github.com/openfun/marsha/pull/1598#discussion_r909700616) 
- I think you should create a component for your custom toast and use an early return
- seconcly, I have a fonctional question:
`When there is a new request to mount on stage, should we alert the teacher only for the new people asking or should we remind him about previous demands ?`
- finaly, if you don't  the toast to appear when the teacher live the tab or any dependency of the useEffect but the array `participants_asking_to_join`, you should not base your return condition on the lenght of the array but rather store it in a ref and test the value stored in the ref with the current value to prompt the toast only when you detect changes

Reply:
@lunika